### PR TITLE
initial trial for motor motion commands

### PR DESCRIPTION
**Pull request by jeriesar94**
This is an initial attempt to write the motor motion commands. It supports moving forwards, backwards, turning left or right, and stopping.

### CODE REVIEW

**Comment by mgharbia** [comment]
[Motors motion commands](https://trello.com/c/JaRXJPuZ/1-motors-motion-commands)

File: motor/scripts/motormotioncommands.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	GPIO.output(RIGHT1,GPIO.LOW)
+	GPIO.output(LEFT1, GPIO.LOW)
+	GPIO.output(RIGHT2, GPIO.HIGH)
+	GPIO.output(LEFT2, GPIO.HIGH)
+
+def turnright():
+	# Note: Output arrangement is important to make sure there is no jitter in the motors when turning Right/Left
+	# TODO: explain the reason for the arrangement in detail in the documentation. 
+	GPIO.output(LEFT1, GPIO.HIGH)
+	GPIO.output(RIGHT2, GPIO.LOW)
```
Comment (mgharbia) [inline motor/scripts/motormotioncommands.py]: The vehicle is tank shaped, I think it's better to set right motor to reverse to make a circle turn right.
Same goes for turnleft()

File: motor/scripts/motormotioncommands.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	GPIO.output(RIGHT1, GPIO.LOW)
+	GPIO.output(LEFT2, GPIO.LOW)
```
Comment (mgharbia) [inline motor/scripts/motormotioncommands.py]: For better readability, I think we should re-arrange the statements in this method to be aligned with moveforward() & movebackward() => meaning to set RIGHT1, LEFT1, RIGHT2, LEFT2  in same order as in moveforward()

File: motor/scripts/motormotioncommands.py (hunk 1 above)
Comment (jeriesar94) [reply motor/scripts/motormotioncommands.py]: That's a good point! This is resolved now.

File: motor/scripts/motormotioncommands.py (hunk 2 above)
Comment (jeriesar94) [reply motor/scripts/motormotioncommands.py]: I also rearranged the statements.

File: motor/scripts/motormotioncommands.py (hunk 1 above)
Comment (mgharbia) [reply motor/scripts/motormotioncommands.py]: I meant moving right motor in reverse while still moving the left forward :)   but again, it is fine,, let's push this and we can change it in real time

Comment (jeriesar94) [reply motor/scripts/motormotioncommands.py]: I think moving one motor in a certain direction and leaving the other one in place would provide a more predictable turning motion, where there will be a pivot point. If both moved, and for some reason there was a difference in speed then the vehicle wouldn't rotate in a circle.

**Pull request merged**